hydra-node/src/Hydra/API/ServerOutput.hs

```diff
@@ -115,3 +121,56 @@ instance
     Greetings me -> Greetings <$> shrink me
     PostTxOnChainFailed p e -> PostTxOnChainFailed <$> shrink p <*> shrink e
     RolledBack -> []
+
+-- | Possible transaction formats in the api server output
+data TxDisplay
```

Comment:
It's not really a display, is it? Also it's not strictly about transactions. Maybe we call this `OutputFormat` with constructor names `OutputAsCBOR` and `OutputAsJSON`?